index.js

```diff
@@ -3,19 +3,8 @@ const express = require("express");
 const app = express();
 const PORT = process.env.PORT || 3000;
 
-/*
-app.get(/\.[a-z0-9]+/i, (req, res) => {
-  res.sendFile(path.join(__dirname, "dist", req.originalUrl));
-});
-
-app.use("/images", express.static(path.join(__dirname, "/dist/images")));
-
-app.get("/*", (req, res) => {
-  res.sendFile(path.join(__dirname, "/dist/index.html"));
-});
-*/
 app.get(/login|register|profile|chat|400|500/, (req, res) => {
```

Comment:
Лучше сделать 
```suggestion
app.get("*", (req, res) => {
```
И обрабатывать на клиенте, так как по невалидному роуту должна отдаться страница 404